leaflet-mapbox-gl.js

```diff
@@ -94,7 +98,12 @@
         getContainer: function () {
             return this._container;
         },
-
+        
+        // returns the pane name set in options if it is a valid pane, defaults to tilePane
+        getPaneName: function () {
+            return this._map.getPane(this.options.pane) ? this.options.pane : 'tilePane'; 
```

Comment:
if the plugin now sets `options.pane`, is the ternary here just a way of being extra careful?

Reply:
The ternary is there to ensure the pane set in options is really present. It is possible for the user to present a pane in the options.pane that is not valid. 

A solution could be to print a warning to the console and return the default pane. I guess that is up to you the maintainer, how much complexity to add.

``` javascript
  // Alternate solution with warning
  // return the pane name if it is valid, prints a warning and returns the default if no pane found
  getPaneName: function () {
    if (!this._map.getPane(this.options.pane)) {
      console.warn(`there is no pane with the name ${this.options.pane}`); 
      return 'tilePane';
    }
    return this.options.pane;
  }
```

Reply:
what you already proposed works for me. thanks for taking the time to explain your rationale.